modules/UI/welcome_page/WelcomePage.js

```diff
@@ -1,68 +1,20 @@
-/* global $, interfaceConfig, APP */
-var animateTimeout, updateTimeout;
-
-var RoomnameGenerator = require("../../util/RoomnameGenerator");
-import UIUtil from "../util/UIUtil";
+/* global $ */
 
 function enter_room() {
```

Comment:
Naming such as `enter_room` is not consistent with our conventions. Unless there's a strong reason against it, please rename it in accord with our naming conventions.